android/routes/ui/src/main/java/com/trafi/routes/internal/LocationDisplayText.kt

```diff
@@ -1,4 +1,4 @@
-package com.trafi.routes.ui.internal
+package com.trafi.routes.internal
```

Comment:
Could you please clarify why you decided to change the package of these files? They are directly related to routes UI.

Reply:
@justasm I just thought that file in internal folder like ViewModel type clases are not directly related with ui, but if it is, then okay, I will leave it

Reply:
I'd leave them there 🙂 They are related to UI, just not meant for SDK users to use directly.